build.gradle

```diff
@@ -288,18 +288,18 @@ dependencies {
     implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: tomcatEmbedVersion
     implementation group: 'org.apache.xmlgraphics', name: 'batik-svgbrowser', version: '1.14'
     implementation group: 'org.apache.xmlgraphics', name: 'xmlgraphics-commons', version: '2.6'
-    implementation group: 'org.springframework', name: 'spring-beans', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-aop', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-aspects', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-context', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-core', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-expression', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-jcl', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-jdbc', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-orm', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-tx', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-web', version: '5.3.19'
-    implementation group: 'org.springframework', name: 'spring-webmvc', version: '5.3.19'
+    implementation group: 'org.springframework', name: 'spring-beans', version: '5.3.20'
+    implementation group: 'org.springframework', name: 'spring-aop', version: '5.3.20'
+    implementation group: 'org.springframework', name: 'spring-aspects', version: '5.3.20'
+    implementation group: 'org.springframework', name: 'spring-context', version: '5.3.20'
+    implementation group: 'org.springframework', name: 'spring-core', version: '5.3.20'
+    implementation group: 'org.springframework', name: 'spring-expression', version: '5.3.20'
```

Comment:
For the Spring version - you could also put the version inside ext and then reference it (see how tomcatEmbedVersion and log4j is implemented).

Reply:
done